star_distribution.py

```diff
@@ -0,0 +1,119 @@
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+
+# define the rotation curve
+def vel(r1):
+
+    ar1 = math.fabs(r1)
+    if (ar1 <= 5.0):
+        v1 = 1.4*ar1*math.exp(-ar1/20.0)
+    elif (ar1 > 5.0 and ar1 < 6.0):
+        max = 1.4*5*math.exp(-5.0/20.0)
+        min = 5.0
+        m = (max - min)/(-1.0)
+        v1 = m*(ar1 - 5.0) + max
+    else:
+        v1 = 5.0
+
+    return v1
+
+def vel(r2):
+
+    ar2 = math.fabs(r2)
+    if (ar2 <= 5.0):
+        v2 = 1.4*ar2*math.exp(-ar2/20.0)
+    elif (ar2 > 5.0 and ar2 < 6.0):
+        max = 1.4*5*math.exp(-5.0/20.0)
+        min = 5.0
+        m = (max - min)/(-1.0)
+        v2 = m*(ar2 - 5.0) + max
+    else:
+        v2 = 5.0
+
+    return v2
+
+def vel(r3):
+
+    ar3 = math.fabs(r3)
+    if (ar3 <= 5.0):
+        v3 = 1.4*ar3*math.exp(-ar3/20.0)
+    elif (ar3 > 5.0 and ar3 < 6.0):
+        max = 1.4*5*math.exp(-5.0/20.0)
+        min = 5.0
+        m = (max - min)/(-1.0)
+        v3 = m*(ar3 - 5.0) + max
+    else:
+        v3 = 5.0
+
+    return v3
+
```

Comment:
It looks like the three above functions perform the exact same operation. Instead, define a function, then call it and apply it to the different argument three times.

Ex: 
def vel(r):
   ...
    function body
   ...
   return v

v1=vel(r1)
v2=vel(r2)
 and so on